ui/00_message/tutorial_guide/tutorial_guide.xml

```diff
@@ -417,10 +417,12 @@
   <message>
     <key>TUTORIAL_175</key>
     <original>戦徒の契り</original>
+    <translation>Right of Pledge</translation>
   </message>
   <message>
     <key>TUTORIAL_176</key>
     <original>着飾り装備</original>
+    <translation>Costumes</translation>
```

Comment:
Yeah this and a few others where changed but I made a mistake somewhere and where reverted